ops/cli.py

```diff
@@ -1,351 +1,309 @@
-        set_up_k8s(project=project, namespace=args.namespace, cluster=cluster, zone=args.zone)
-
-        version = shell("cat version")
-        publish_docker_images(version=version)
-
-        deploy = os.path.join("derived", "helm", "deploy")
-        if not os.path.exists(deploy):
-            os.makedirs(deploy)
-
-        values = "wfl-values.yaml"
-        ctmpl = f"derived/2p/gotc-deploy/deploy/{gotc_folder}/helm/{values}.ctmpl"
-        shutil.copy(ctmpl, deploy)
-
-        render_ctmpl(ctmpl_file=f"{deploy}/{values}.ctmpl", WFL_VERSION=version)
-        helm_deploy_wfl(values=f"{deploy}/{values}")
-
-        container = run_cloudsql_proxy(project=project, cloudsql_instance_name="zero-postgresql")
-        with open(f"{deploy}/{values}") as f:
-            helm_values = yaml.safe_load(f)
-            env = helm_values['api']['env']
-            run_liquibase_migration(env['ZERO_POSTGRES_USERNAME'], env['ZERO_POSTGRES_PASSWORD'])
-            info("=> Stopping cloud_sql_proxy")
-            shell(f"docker stop {container}")
-
-        success("[✔] Deployment is done!")
-        shell("kubectl get pods")
+    shell(f"docker run --rm --net=host "
+          f"-v {changelog_dir}:/liquibase/changelog liquibase/liquibase "
+          f"--url='{db_url}' --changeLogFile=/changelog/changelog.xml "
+          f"--username='{config.db_username}' --password='{config.db_password}' update")
+    success("Ran liquibase migration")
+
+
+def stop_cloud_sql_proxy(config: WflInstanceConfig) -> None:
+    """Stop the stored Cloud SQL proxy container."""
+    info("=>  Stopping cloud_sql_proxy")
+    shell(f"docker stop {config.cloud_sql_local_proxy_container}")
+
+
+def print_deployment_success(config: WflInstanceConfig) -> None:
+    """Print success and get pods from existing kubectl context."""
+    success(f"{config.command} is done!")
+    info(shell("kubectl get pods"), plain=True)
+
+
+command_mapping: Dict[str, List[Callable[[WflInstanceConfig], None]]] = {
+    "info": [
+        infer_missing_arguments_pre_validate,
+        validate_cloud_sql_name,
+        validate_cluster_name,
+        infer_missing_arguments,
+        print_config
+    ],
+    "connect": [
+        infer_missing_arguments_pre_validate,
+        validate_cloud_sql_name,
+        infer_missing_arguments,
+        print_config,
+        exit_if_dry_run,
+        configure_cloud_sql_proxy,
+        print_cloud_sql_proxy_instructions
+    ],
+    "deploy": [
+        infer_missing_arguments_pre_validate,
+        validate_cloud_sql_name,
+        validate_cluster_name,
+        infer_missing_arguments,
+        print_config,
+        configure_kubectl,
+        configure_helm,
+        render_values_file,
+        exit_if_dry_run,
+        prompt_deploy_version,
+        configure_cloud_sql_proxy,
+        publish_docker_images,
+        helm_deploy_wfl,
+        run_liquibase_migration,
+        stop_cloud_sql_proxy,
+        print_deployment_success
+    ]
```

Comment:
@ehigham and @rexwangcc both mentioned they like to comment out parts of this script in order to make it do whatever specific thing they want. I think this part will work well for that: there's obviously some interdependencies but other than that things can be removed or reordered with impunity. Also becomes very easy to make new commands in the same vein.

Reply:
Also, I specifically configure the Cloud SQL proxy before any of the tasks with remote side effects because it is the most likely to fail. Now if you get a port error you can just clear the port and deploy again without needing to comment anything out.

Reply:
I like this change!